### DISCUSSION TITLE
What happened? Why I cannot call the button which defined in other notes?

### DISCUSSION THREAD

**Discussion by francisHugo** [mProjectsCode/obsidian-meta-bind-plugin#261] [Q&A] [answered]
![image](https://github.com/mProjectsCode/obsidian-meta-bind-plugin/assets/129242259/beaa75ed-28f8-483d-ae84-b7cfa4fdcac5)

**Comment by mProjectsCode** [comment, marked as answer]
> Inline buttons must reference a button code block defined elsewhere in the same note via matching ids.

See [the docs](https://www.moritzjung.dev/obsidian-meta-bind-plugin-docs/guides/buttons/#inline-buttons).

Buttons defined in the plugin's settings are available via their ID in all notes.

**Comment by francisHugo** [comment]
Oh, I understood now.

I used to use the plugin `button` , with the plugin,  buttons defined in a note (they have  IDs), can be used in another note. 

I had put all the buttons which I created in a note named `button`, then I call (reuse) the button anywhere in the whole vault.

Will the `meta bind` button do the same?

**Reply by mProjectsCode** [reply]
Like I said, if you define the buttons in the plugin's settings under `Button Templates` then you can access them via the id in every note.
Like I have answered in other discussions already, button ids being scoped to the file that they are in has technical reasons and won't be changed.

**Reply by francisHugo** [reply]
Thank you very much for your answer. I got it. 

I looked the `Button Templates` before but do not know what it is for, now, I understood.

thank you for the great plugin.
